doc/sample-problems/DiffEq/HeavisideStep.pg

```diff
@@ -0,0 +1,117 @@
+## DESCRIPTION
+## Adding the Heaviside function to the context.
+## ENDDESCRIPTION
+
+## DBsubject(WeBWorK)
+## DBchapter(WeBWorK tutorial)
+## DBsection(PGML tutorial 2015)
+## Date(06/01/2015)
+## Institution(Hope College)
+## Author(Paul Pearson)
+## MO(1)
+## KEYWORDS('differential equations', 'adding the Heaviside function to the context')
+
+#:% name = Heaviside or Step Function
+#:% type = Sample
+#:% subject = differential equations
+#:% categories = [heaviside/step]
```

Comment:
Since these categories are used for html id's they can not have forward slashes in them.  Those are invalid characters for id's.  So this will need to be `heaviside-step`.

Reply:
okay